docs/csharp/language-reference/compiler-messages/cs8795.md

```diff
@@ -0,0 +1,62 @@
+---
+description: "Compiler Error CS8795"
+title: "Compiler Error CS8795"
+ms.date: 9/29/2023
+f1_keywords:
+  - "CS8795"
+helpviewer_keywords:
+  - "CS8795"
+---
+# Compiler Error CS8795
+
+Partial method 'method' must have an implementation part because it has accessibility modifiers.
+
+## Example
+
+ The following sample generates CS8795:
+
+```csharp
+// CS8795.cs (4,25)
+
+public partial class PartialClass
+{
+    public partial void PartialMethod();
+}
+```
+
+## To correct this error
+
+Add an implementation in another partial class:
+
+```csharp
+public partial class PartialClass
+{
+    public partial void PartialMethod();
+}
+
+public partial class PartialClass
+{
+    public partial void PartialMethod() {}
+}
+```
+
+> [!NOTE]
+> The following is not allowed (having both declaration and implementation together):
+>
+> ```csharp
+> public partial class PartialClass
+> {
+>     public partial void PartialMethod() {}
+> }
+> ```
+>
```

Comment:
I think this is confusing. 

If I understand correctly, this note is meant to show that you must have one partial declaration without an implementation. The example doesn't make that clear, because it looks exactly like the previous (valid) example. 

Here's one suggestion. If you have better wording, or if I misinterpreted what you were going for, make a different edit:

```suggestion
> [!NOTE]
> A partial method must have exactly one declaration without a body. A partial method with an access modifier *must* have exactly one declaration that includes the implementation.
>
> ```csharp
> public partial class PartialClass
> {
>     public partial void PartialMethod() {}
> }
> ```
>
```